src/utils.ts

```diff
@@ -1,12 +1,19 @@
 import Metalsmith from 'metalsmith';
 import match from 'multimatch';
 import path from 'path';
+import pug from 'pug';
+
+import { DeepReadonly } from './utils/deep-readonly';
```

Comment:
I think that `src/utils/deep-readonly.ts` is a file name specialized for `DeepReadonly` type.
I think it is better to change to a generic name like `src/utils/types.ts`.

Reply:
If we change to a type definition utility file, I think it is better to include [the `isAnyArray` type](https://github.com/sounisi5011/metalsmith-pug-extra/blob/db0e19d0d1d6237f0735da6c583c39c379e65387/src/utils.ts#L13-L15).